editor/scene_tree_dock.cpp

```diff
@@ -2210,7 +2211,7 @@ void SceneTreeDock::_create() {
 		_do_reparent(last_created, -1, nodes, true);
 	}
 
-	scene_tree->get_scene_tree()->call_deferred("grab_focus");
+	scene_tree->get_scene_tree()->grab_focus();
```

Comment:
I tested and it seems to work the same without the `call_deferred()`

Reply:
CC @KoBeWi who added it as deferred in 9432e06e056e3272662fa8e0fdf7f7c227b66b1b.
Did you check that it works also for e.g. node reparenting?

Reply:
I made it deferred, because it didn't do anything otherwise. A lots of code changed since then, so maybe it's not needed anymore.